pkg/components/azure-arc-onboarding/image/azure-arc.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Type of action: register or remove the Lokomotive cluster from Azure Arc.
+action=$1
+
+function generate_kubeconfig() {
+  ca="$(cat /var/run/secrets/kubernetes.io/serviceaccount/ca.crt | base64 | tr -d "\n")"
+  namespace="$(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace)"
+  token="$(cat /var/run/secrets/kubernetes.io/serviceaccount/token | tr -d "\n")"
+  server="https://kubernetes.default.svc"
+
+  echo "
+  apiVersion: v1
+  kind: Config
+  clusters:
+  - name: default-cluster
+    cluster:
+      certificate-authority-data: ${ca}
+      server: ${server}
+  contexts:
+  - name: default-context
+    context:
+      cluster: default-cluster
+      namespace: ${namespace}
+      user: default-user
+  current-context: default-context
+  users:
+  - name: default-user
+    user:
+      token: ${token}
+  " > sa.kubeconfig
```

Comment:
Do we need kubeconfig? We can use the token directly, right? As documented [here](https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/cluster-connect#option-2-service-account-bearer-token):

```
az connectedk8s proxy -n <cluster-name> -g <resource-group-name> --token $TOKEN
```

Reply:
`az connectedk8s connect` doesn't have this option.